domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/IsBidirectional.java

```diff
@@ -0,0 +1,26 @@
+package org.mobilitydata.gtfsvalidator.domain.entity;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.stream.Stream;
+
+public enum IsBidirectional {
```

Comment:
Replace by a boolean

Reply:
We may want to keep this as an `enum` to allow future flexibility. It's possible this GTFS field could be overloaded with another value in the future, even though it seems like it should only have two possible values.